app/models/beatmap.py

```diff
@@ -113,6 +115,7 @@ def to_dict(self) -> dict[str, Any]:
             "ranked_status_freezed": self.frozen,
             "rankedby": self.rankedby,
             "rating": self.rating,
+            "bancho_ranked_status": self.bancho_ranked_status,
         }
```

Comment:
Hmm this raised a question in my mind of whether this should int() cast, but seems this .to_dict() is never actually used